lib/enrollment.rb

```diff
@@ -1,35 +1,10 @@
-require 'pry'
-require_relative 'district'
 class Enrollment
-  attr_accessor :name, :school_data #:high_school_graduation_rates,
+  attr_accessor :name, :high_school_graduation_rates
+  attr_reader :kindergarten_participation
 
-  def initialize(enrollment_data)
-    @name = enrollment_data[:name]
-
-    @school_data = {kindergarten_participation: enrollment_data[:kindergarten_participation], high_school_graduation_rates: enrollment_data[:high_school_graduation_rates]}
-  end
-
-  def kindergarten_participation
-    school_data[:kindergarten_participation]
-  end
-
-  def high_school_graduation_rates
-    school_data[:high_school_graduation_rates]
-  end
-
-  def kindergarten_participation_in_year(year = nil)
-    if year
-      school_data[:kindergarten_participation][year]
-    else
-      school_data[:kindergarten_participation]
-    end
-  end
-
-  def graduation_rate_by_year
-    high_school_graduation_rates
-  end
-
-  def graduation_rate_in_year(year)
-    high_school_graduation_rates[year]
+  def initialize(data)
```

Comment:
I think school_data is an unnecessarily a hash structure because it'll end up containing more hashes, adding unnecessary computational complexity to our application.